expyfun/stimuli/tests/test_tracker.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerBinom, TrackerDealer
+from expyfun import ExperimentController
+from nose.tools import assert_raises
+import matplotlib.pyplot as plt
```

Comment:
I think you can fix the CircleCI error by inserting `import matplotlib; matplotlib.use('Agg')` _above_ this line that imports `pyplot`.  See [here](https://github.com/LABSN/expyfun/blob/master/expyfun/analyze/tests/test_viz.py#L11)